readme.md

```diff
@@ -168,6 +168,7 @@
 - [Material Design](https://github.com/sachin1092/awesome-material)
 - [REST](https://github.com/marmelab/awesome-rest)
 - [Malware Analysis](https://github.com/rshipp/awesome-malware-analysis)
+- [Awesome Gists](https://github.com/vsouza/awesome-gists) 
```

Comment:
Do you see any of the other entries including the `Awesome` prefix? Hint hint ;)